src/components/Nav/Nav.js

```diff
@@ -1,14 +1,33 @@
 import React, { useState } from 'react';
 import * as N from './Nav.styles';
 import { useNavigate } from 'react-router-dom';
+import { BiBookmark } from 'react-icons/bi';
+import { MdNotificationsNone } from 'react-icons/md';
+import { BsCart2 } from 'react-icons/bs';
+import Search from '../Search/Search';
 
 function Nav() {
+  const [write, setWrite] = useState(false);
   const navigate = useNavigate();
+  const token = localStorage.getItem('token');
+
+  const delToken = name => {
+    if (name === '로그아웃') {
+      localStorage.removeItem('token');
+      window.location.reload();
+    }
+  };
```

Comment:
```suggestion
 const handleLogin = () => {
    if (token) {
      localStorage.removeItem('token');
      window.location.reload();
    } else {
      navigate('/login');
    }
  };
```

로그인과 로그아웃을 하나의 버튼에서 구현되도록 했을 때
이 함수도 토큰에 따라서 다른 동작을 하도록 구현하는 것이 좋겠습니다.

Reply:
넵 전체 반영했습니다.